docs/modules/user-guide/partials/proc_adding-event-bindings.adoc

```diff
@@ -2,16 +2,16 @@
 = Adding event bindings
 
 [role="_abstract"]
-Adding event bindings.
+This section describes how to add an event to devfiles v2.x. An event specified in a devfile v1.x still works in a devfile v2.x.
 
 .Prerequisites
 
-
+Add an `events` section in your devfile and make sure it contains a list of `preStart` and `postStart` commands.
 
 .Procedure
 
 . Add an `events` section in the devfile, containing a list of `preStart` and `postStart` commands.
```

Comment:
to your devfile

Reply:
Here I see the same sentence twice. It can only be either a prerequisite or a procedure.